components/layout.tsx

```diff
@@ -36,7 +36,7 @@ export default function Layout({
           <>
             <Image
               priority
-              src="/images/profile.jpg"
```

Comment:
was the file rename needed?
you did it here and also renamed the file. if you hand't changed any of them the result should have been the same